packages/flutter_tools/test/commands.shard/permeable/packages_test.dart

```diff
@@ -249,7 +219,6 @@ void main() {
         usage: globals.flutterUsage,
         botDetector: globals.botDetector,
         platform: globals.platform,
-        stdio: mockStdio,
```

Comment:
Ahh, so the tests in this file now spit out the pub stdout to the terminal stdout, where before they were mocked out. Let me think if there's a way we can avoid that...